onnxruntime/core/providers/cpu/reduction/reduction_ops.h

```diff
@@ -54,6 +55,7 @@ constexpr TensorOpCost ParallelReduceFastCost(int64_t n_row, int64_t n_col, int6
   *  KR - reduction on the last dimensions
   *  RK - reduction on the first dimensions
   *  KRK - reduction on the middle dimensions.
+  *  RKR - reduction on the middle dimensions on all dimensions but the middle one
 
```

Comment:
nit: should this just say 'reduction on all dimensions but the middle one'?

Reply:
done